tests/test-cases/Group1-Docker-Commands/1-39-Docker-Stats.md

```diff
@@ -0,0 +1,32 @@
+Test 1-39 - Docker Stats
+=======
+
+#Purpose:
+To verify that `docker stats` is supported and works as expected.
+
+#Environment:
+This test requires that a vSphere server is running and available
+
+
+#Test Steps:
+1. Run a busybox container and create a busybox container
+2. Run Stats with no-stream which will return stats for any running container
+3. Run Stats with no-stream  all which will return stats for all containers
+4. Verify the API memory output against govc
+5. Verify the API CPU output against govc
```

Comment:
6. Run stats with no-stream on a container that doesn't exist
7. Run stats with no-stream \<running container ID\>
8. Run stats with no-stream \<non-running container ID\>